src/cljs/witan/ui/data.cljs

```diff
@@ -84,6 +84,33 @@
    :kixi.datastore.metadatastore/bundle-add
    ])
 
+(def metadata-list-fields
+  [:kixi.datastore.metadatastore/name
+   :kixi.datastore.metadatastore/id
+   [:kixi.datastore.metadatastore/provenance
+    :kixi.datastore.metadatastore/created]
+   [:kixi.datastore.metadatastore/provenance
+    :kixi.user/id]
+   :kixi.datastore.metadatastore/type
+   :kixi.datastore.metadatastore/bundle-type
+   :kixi.datastore.metadatastore/file-type
+   :kixi.datastore.metadatastore/license
+   :kixi.datastore.metadatastore/size-bytes
+   :kixi.datastore.metadatastore/sharing])
+
+(def search-dashboard-default
+  {:query {:kixi.datastore.metadatastore.query/name {:match ""}}
+   :size 50
```

Comment:
The downside of putting constants like this into the app state is that we can't update them without forcing people to log out and log back in. Not ideal but possibly tricky to work into the state otherwise in this case.

Reply:
In my imagination this will be changed by the user via a page size control. The idea is that these queries get manipulated via various controls and then executed. These are just the starter values, rather than defaults as such.